src/palette/qml/MuseScore/Palette/PaletteCellPropertiesDialog.qml

```diff
@@ -84,7 +84,7 @@ StyledDialogView {
                         //: Abbreviation of "spatium"
                         measureUnit: qsTrc("global", "sp") },
                     { title: qsTrc("palette", "Y"), value: propertiesModel.yOffset, incrementStep: 1, measureUnit: qsTrc("global", "sp") },
```

Comment:
You also need to add a `minValue` entry for "X" and "Y" too, because otherwise, for those properties, the value of `modelData["minValue"]` will be `undefined`, which is converted to 0 when assigning it to the `minValue` property of the spin box. But it should not be 0, it should be -999, so that needs to be specified in the model.

Reply:
Currently it seems to be using the default value of -999 instead of 0 but I will add it to the others for the sake of clarity.

Reply:
Yes, IncrementalPropertyControl uses -999 by default if you don't specify anything, but if you do specify something that turns out to evaluate to `undefined`, that will be converted to `0`, so will have the same effect as specifying `0`, which overrides the `-999`.